mlrun/__main__.py

```diff
@@ -902,6 +909,7 @@ def project(
     env_file,
     timeout,
     ensure_project,
+    schedule,
```

Comment:
If the flag name is `--schedule-pipeline` I think the variable name should be `schedule_pipeline` meaning this is not really working, you had a lot of comments on this PR, meaning a lot of changes, please make sure to test everything after all changes - ideally obviously to just write tests so it's automatic

Reply:
i think it should be `--schedule` since its obvious we talk about a pipeline

Reply:
lets keep it`--schedule` this function is for pipelines

Reply:
back to `--schedule`